arangod/Aql/ExecutionEngine.cpp

```diff
@@ -383,15 +383,9 @@ struct CoordinatorInstanciator : public WalkerWorker<ExecutionNode> {
           break;
       }
     } else {
-#ifdef USE_IRESEARCH
-      if (ExecutionNode::ENUMERATE_IRESEARCH_VIEW == nodeType) {
-        return false;
-      }
-#endif
-
       // on dbserver
       _dbserverParts.addNode(en);
-      if (nodeType == ExecutionNode::REMOTE) {
+      if (ExecutionNode::REMOTE == nodeType) {
```

Comment:
why use yoda style ?

Reply:
that usually helps me avoid stupid mistakes